frontend/src/components/navigation/menuItem.tsx

```diff
@@ -1,44 +1,44 @@
-import {Menu as AMenuItem} from "antd";
-import styled from "styled-components";
-import {ProfileFilled} from '@ant-design/icons';
+import { Menu } from 'antd';
+import { Link } from 'react-router-dom';
+import styled from 'styled-components';
 
+const Container = styled.div`
 
- const Icon = styled(ProfileFilled)
- `
- font-size: 18px;
- width: 1em;
-    height: 1em;
-    overflow: hidden;
-    font-size: 1.5rem;
-    vertical-align: middle;
-    font-family: 'Material Icons';
-    margin-right: 10%;
-    margin-left: 10%;
-    line-height: 1;
-    letter-spacing: normal;
- `;
-const MenuItem = styled(AMenuItem)`
+ background-color:inherit;
+ color: #bdbfc7;
+ font-family: Arial;
+ font-size: 14px;
+ width: 100%;
+ height: 100vh;
+ width: 260px;
+ padding-top: 17px;
+ padding-bottom:17px;
+ border-right-width: 0px;
+ display: flex;
+ align-items: center;
+ p{
+  padding-left: 20px;
+  margin-bottom:0px;
+ }
+ span{
+    margin-right: 20px; 
+   }
+     &:hover {
+       background-color: #1a1e31;
+       color: white;
```

Comment:
Tutaj to samo, stała `Colors`, jeśli nie ma tam jakiegoś koloru to po prostu wystarczy utworzyć nowy